src/Client.php

```diff
@@ -433,8 +433,8 @@ protected function createPipeline(array $options = null, $callable = null)
             $class = 'Predis\Pipeline\Pipeline';
         }
 
-        /*
-         * @var ClientContextInterface
+        /**
```

Comment:
Yup that was just an oversight on my part, my doubt was about the addition of `$pipeline`.